CONTRIBUTING.md

```diff
@@ -50,7 +42,7 @@ We keep a running list of changes to this application in the [CHANGELOG](CHANGEL
   - [React Router](https://github.com/ReactTraining/react-router/tree/v3/docs)
 
 ### Workflow
-- Open branches off main repo due to secure CircleCI env vars.
+- Open branches off main repo (as opposed to a fork) due to secure CircleCI env vars.
```

Comment:
Existing issue, but can we make this clearer? Ex: Open a branch (not a fork) from master in the [main repo](/link). This keeps our CircleCI integration from ...